src/paperless/settings.py

```diff
@@ -109,7 +109,7 @@ def __get_boolean(key, default="NO"):
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
 
 # We allow CORS from localhost:8080
-CORS_ORIGIN_WHITELIST = tuple(os.getenv("PAPERLESS_CORS_ALLOWED_HOSTS", "localhost:8080").split(","))
+CORS_ORIGIN_WHITELIST = tuple(os.getenv("PAPERLESS_CORS_ALLOWED_HOSTS", "http://localhost:8080").split(","))
```

Comment:
How is this related to upgrading the operating system?

Reply:
Sorry, forgot to mention this in the commit message. After upgrading the packages in the lock file, some upgraded dependency for cors now requires full URIs instead of hostnames now.

Reply:
Okay, thanks for clarifying :+1:

Reply:
Can we mention this setting in [the docs](https://github.com/the-paperless-project/paperless/blame/master/docs/setup.rst#L196)? I would expect people having SSL enabled to change this. Or can we get away with `//` instead of `http://` to support both?

Reply:
I think a proper URL is required now. So // probably doesn't work.

Reply:
You could just add both URLs, couldn't you?

Reply:
done